### PR TITLE
Stream

### PR DESCRIPTION
**Pull request by anandsr21harshit**
- Added Video Streaming Page
- User can now click on video thumbnail and it will open a new page
- ReactPlayer is used
- Fully responsive layout

### CODE REVIEW

**Comment by anandsr21harshit** [comment]
Check live preview here:
https://lsytjn.csb.app/

**Review by thegeorgenikhil — COMMENTED** [review]
Really good responsive design🔥

File: src/frontend/pages/Stream/Stream.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+function Stream() {
+  const { videoID } = useParams();
+  const {state} = useData();
+  const {videos} = state;
+
+  function findVideo(id){
+      const currentVideo = videos.find(video => video._id === id)
+      return currentVideo;
+  }
+
```
Comment (thegeorgenikhil) [inline src/frontend/pages/Stream/Stream.jsx:17]: This can be converted to an arrow function

**Review by thegeorgenikhil — APPROVED** [review]
Functionality works as expected.
Impressive UI.
Ready to merge.

File: src/frontend/pages/Stream/Stream.jsx (hunk 1 above)
Comment (anandsr21harshit) [reply src/frontend/pages/Stream/Stream.jsx:17]: well I prefer this way

**Comment by sushmita12345** [comment]
- Appreciate your work.
- Fully responsive as expected.
- All functionality work very well, you can proceed to merge.

**Review by sushmita12345 — APPROVED** [review]

**Pull request merged**